src/pages/fr/core-concepts/project-structure.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Structure du Projet
+description: Apprends à structurer un projet avec Astro.
+---
+
+Votre tout nouveau projet Astro généré à partir de l'assistant de création `create-astro` a déjà quelques des fichiers et dossiers. Les autres fichiers, vous allez les créer vous-même et ajouter à la structure de fichiers d'Astro.
```

Comment:
"...et les ajouter..."? I'm not sure if it's needed here, or if it's common style in French to add it.